manifest.json

```diff
@@ -5,41 +5,41 @@
     "name": "Share Backported",
     "version": "2.0.1",
     "icons": {
-        "16": "icon-dark.svg",
-        "24": "icon-dark.svg",
-        "48": "icon-dark.svg",
-        "96": "icon-dark.svg"
+        "16": "icon.svg",
+        "24": "icon.svg",
+        "48": "icon.svg",
+        "96": "icon.svg"
     },
     "browser_action": {
         "browser_style": true,
         "default_icon": {
-            "16": "icon-dark.svg",
-            "24": "icon-dark.svg",
-            "48": "icon-dark.svg",
-            "96": "icon-dark.svg"
+            "16": "icon.svg",
+            "24": "icon.svg",
+            "48": "icon.svg",
+            "96": "icon.svg"
         },
         "theme_icons": [{
-            "dark": "icon-dark.svg",
-            "light": "icon-light.svg",
+            "dark": "icon.svg",
+            "light": "z.svg",
```

Comment:
this file doesn't exist.
Please revert everything as the original manifest and filesnames

Reply:
Oh, it's a typo. I'm so sorry